lang/fr-FR/display_guide_vocabulary_edrlab.json

```diff
@@ -0,0 +1,351 @@
+{
+    "metadata": {
+        "author": "The European Digital Reading Lab (EDRLab)",
+        "language": "fr-FR",
+        "variant": "canonical",
+        "version": "2.0.c",
+        "audience": "general",
+        "description": "Established collectively during workshops with 30 organisations representing the ecosystem, tested with users thru surveys and prototype implementation"
```

Comment:
Should the description be also in French?
@gregoriopellegrino Italian version was in Italian. (thru should be through)

Reply:
Good question. This key is here to help implementers make a choice, so I would expect it in English.

Reply:
If we agree then, @gregoriopellegrino  we should update the Italian version to have that description in English as well.